utils/testData.rb

```diff
@@ -9,23 +9,15 @@ def initialize(library)
   end
 
   def loading_data(library)
-    author1 = Author.new('Gregory Brown',
-                         'Has run the independently published Practicing Ruby journal since 2010,
-                                   and is the original author of the popular Prawn PDF generation library.')
-    author2 = Author.new('David Chelimsky',
-                         'Developer and lead cavaquinista at @cognitect.')
-    author3 = Author.new('Paolo Perrotta',
-                         'Has more than ten years of experience as a developer and writer')
-    author4 = Author.new('Peter J. Jones',
-                         'Peter J. Jones has been working professionally with Ruby since 2005')
-    author5 = Author.new('Leonard Richardson',
-                         'Author of robotfindskitten,Beautiful Soup,RESTful Web APIs,and Constellation Games')
-    author6 = Author.new('Jay McGavren',
-                         'Jay is a development teacher at Treehouse, and an occasional conference speaker')
-    author7 = Author.new('Dawn Griffiths',
-                         'Popular Series By Dawn Griffiths')
+    author1 = Author.new('Gregory Brown', 'Has run the independently published Practicing Ruby')
```

Comment:
Create array constant which includes hashes of authors data:
```ruby
[
  { name: 'Gregory Brown',  biography: 'Has run the independently published Practicing Ruby' },
  ...
]
```
Then use `array.each` and `library.authors << Author.new(author_data)`